tests/tests_API_FB.py

```diff
@@ -0,0 +1,38 @@
+# Import and load necessary lib
+# import sys
+# sys.path.append('vagrant\ultimatejob')
+import API_FB
+
+
+def main():
+    # Create soup object & print a status message - 404 not good , 200 good
+    soup = API_FB.create_soup(API_FB.create_req())
+
+    # Return list contains URLs
+    API_FB.extract_jobs_url(soup)
+
+    # Return list contains job titles
+    API_FB.extract_jobs_title(soup)
+
+
+# Test functions
+def test_api_urls():
+    """The testing will check if our functions return the required output"""
+    expected_output_urls = ['https://www.facebook.com/careers/jobs/1672813472870915/',
+                            'https://www.facebook.com/careers/jobs/712878282607325/',
+                            'https://www.facebook.com/careers/jobs/420212419368641/',
+                            'https://www.facebook.com/careers/jobs/207984144060781/',
+                            'https://www.facebook.com/careers/jobs/773785633436218/',
+                            'https://www.facebook.com/careers/jobs/2597607417128369/',
+                            'https://www.facebook.com/careers/jobs/323705805605018/',
+                            'https://www.facebook.com/careers/jobs/536078547072736/']
+    assert API_FB.extract_jobs_url() == expected_output_urls, "The tests for URLs were successful"
```

Comment:
You should add the following 
 soap = API_FB.create_soup(API_FB.create_req())
 assert API_FB.extract_jobs_url(soap) == expected_output_urls, "The tests for URLs were successful"

You will see that positions url changed in FB :) test will fail